dali/pipeline/executor/executor.h

```diff
@@ -140,6 +140,71 @@ class DLL_PUBLIC Executor : public ExecutorBase, public WorkspacePolicy, public
 
   void SetupOutputQueuesForGraph();
 
+  template <typename Workspace>
+  void RunHelper(OpNode &op_node, Workspace &ws) {
```

Comment:
Can you merge this two RunHelper into one. They are mostly copy-paste.

Reply:
Done